src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -726,7 +714,7 @@ static bool _toColor(const char* str, uint8_t* r, uint8_t* g, uint8_t* b, char**
         content = str + 4;
         content = _skipSpace(content, nullptr);
         if (_parseNumber(&content, &hue, &th) && hue) {
-            th = static_cast<uint32_t>(th) % 360;
+            th = float(uint32_t(th) % 360);
```

Comment:
There seems to be a problem with the existing source, How about using fmod()?

Reply:
I think you can fix it properly. This pr is just about a code refactoring due to the compiler warnings.